src/index.ts

```diff
@@ -26,15 +70,16 @@ let profiler: Profiler|undefined = undefined;
  * config - Config describing configuration for profiling.
  *
  * @example
- * profiler.start({});
+ * profiler.start();
  *
  * @example
  * profiler.start(config);
  *
  */
-export function start(config: Config): Promise<void> {
+export async function start(config: Config = {}): Promise<void> {
   try {
-    profiler = new Profiler(config);
+    const normalizedConfig = await initConfig(config);
+    profiler = new Profiler(normalizedConfig);
     return profiler.start();
   } catch (e) {
     return Promise.reject(e);
```

Comment:
Same comment as above with regards to returning `Promise.reject`, since `start` is now `async`.